docsite/rst/intro_windows.rst

```diff
@@ -217,6 +217,9 @@ Pass the -CertValidityDays option to customize the expiration date of the genera
 Pass the -SkipNetworkProfileCheck switch to configure winrm to listen on PUBLIC zone interfaces.  (Without this option, the script will fail if any network interface on device is in PUBLIC zone)
   powershell.exe -File ConfigureRemotingForAnsible.ps1 -SkipNetworkProfileCheck
 
+Pass the -ForceNewSSLCert switch to force a new SSL certificate to be attached to and already existing winrm listener. (Avoids SSL winrm errors on syspreped Windows images after the CN changes)
```

Comment:
Thanks for tackling the documentation too.
I know I'm being super-fussy here but I think you mean 'to an already existing' not 'to and already existing'.?